### ISSUE TITLE
Granting permissions to iOS apps

### ISSUE THREAD

**Issue by bartekpacia** [danielpaulus/go-ios#283]
Hi 👋🏻 

I'm curious about whether `go-ios` could support granting permissions to iOS apps, [just like it's possible with `adb` on Android](https://stackoverflow.com/questions/16410167/how-do-i-use-adb-grant-or-adb-revoke).

**Comment by shamanec** [comment]
I don't think you can do it via Mac/Xcode so I doubt it will be possible via `go-ios` as well, but who knows 🤷‍♂️

**Comment by danielpaulus** [comment]
Unfortunately there is no real-device service that can grant or revoke TCC permissions: `xcrun simctl privacy` is simulator-only, and XCTest's `resetAuthorizationStatus` requires a test host process. Nothing equivalent to `adb shell pm grant` exists for physical iOS devices, so this is not something go-ios can implement. Closing as not planned; we would revisit if Apple ever exposes such a service.

_(part of an automated backlog sweep — comment or reopen if this got it wrong)_
